lists/ir.csv

```diff
@@ -979,3 +979,5 @@ https://www.instagram.com/p/BxcElVMhBAH/,HUMR,Human Rights Issues,2019-06-04,che
 https://scontent.cdninstagram.com/vp/3ac38df0bf671f725ac631abb30c526e/5CF92CB9/t50.16885-16/10000000_286983405520127_242845370835533824_n.mp4?_nc_ht=scontent.cdninstagram.com,HUMR,Human Rights Issues,2019-06-04,chelchela,
 https://scontent.cdninstagram.com/vp/1f4c22d2412c022f90c4d06687d8f80e/5CF97FDE/t50.2886-16/57482666_1173844336109860_1640024195301638144_n.mp4?_nc_ht=scontent.cdninstagram.com,HUMR,Human Rights Issues,2019-06-04,chelchela,
 https://instagram.fhen1-1.fna.fbcdn.net/vp/0e7c843867a49565bcffe85263156204/5CF963B6/t51.2885-15/e35/56811531_324081334963144_7890181447627396873_n.jpg?_nc_ht=instagram.fhen1-1.fna.fbcdn.net,HUMR,Human Rights Issues,2019-06-04,chelchela,
+https://termux.org/,HOST,Hosting and Blogging Platforms,2019-08-06,chelchela,
```

Comment:
I guess this is not the most appropriate category, but of all the ones in the list it's probably the most fitting.